### PR TITLE
Abb 18 nv

### PR DESCRIPTION
**Pull request by nafisa-valieva**
Here I have created another stub for a manuscript I referred in other records

### CODE REVIEW

File: ParisBNF/abb/BNFabb18.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10091827s">BnF Éthiopien d'Abbadie 18</idno>
+               </msIdentifier>
+               <msContents>
+                  <summary/>
+
+                  <msItem xml:id="ms_i1">
+                     <locus from="3r"/>
```
Comment (karljonaskarlsson) [inline ParisBNF/abb/BNFabb18.xml:36]: do both LIT3309Qerellos and LIT3310Prosph begin on fol. 3r?

File: ParisBNF/abb/BNFabb18.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <particDesc>
+            <listPerson>
```
Comment (karljonaskarlsson) [inline ParisBNF/abb/BNFabb18.xml:125]: I didn't know about listPerson! It's good to learn something new

**Review by karljonaskarlsson — APPROVED** [review]

File: ParisBNF/abb/BNFabb18.xml (hunk 1 above)
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb18.xml:36]: they do, LIT3310Prosph is part of LIT3309Qerellos. I just need to develop LIT3309Qerellos more, but there is a lot to be done :-)

File: ParisBNF/abb/BNFabb18.xml (hunk 2 above)
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb18.xml:125]: Denis is to be thanked for it!

Comment (eu-genia) [reply ParisBNF/abb/BNFabb18.xml:125]: It is not necessary to list the persons if they are quoted elsewhere with the relevant roles, but it is helpful if there is no other place to mention them, or if the context did not allow to specify each role

**Review by eu-genia — APPROVED** [review]
thank you, ok for a stub, just important not to forget to develop the records

**Pull request merged**